client/src/components/relatedItems/ComparisonModal.jsx

```diff
@@ -8,40 +8,47 @@ class ComparisonModal extends Component {
     super(props);
     this.state = {
     }
+    this.targetOverlay = document.getElementById('overlay');
+    this.uniqueId = "comparison-modal-" + this.props.id;
+    this.uniqueSearch ="#" + this.uniqueId;
+    this.searchString = `[data-modal-target],${this.uniqueId}`;
+    this.openModalButtons = document.querySelectorAll(this.searchString);
+    this.openModalButtonsClick = this.openModalButtonsClick.bind(this);
+
   }
 
+  openModal(modal){
+    if (modal === null) {
+      return;
+    }
+    modal.classList.add('active');
+    this.targetOverlay.classList.add('active');
+  }
+
+  openModalButtonsClick() {
+    console.log('open click');
```

Comment:
Try to remember to remove any non-error console.logs. This will prevent your team from seeing this when they pull down.

Reply:
Hi! The functionality I'm implementing will not render any error messages unless you click on the modal.  And similarly, the console log only happens when the button is clicked... I'll def remove it in the next commit.  Thank you!

Reply:
Sounds good. I will merge. 😄